freeweb/settings.py

```diff
@@ -110,6 +110,11 @@
     SESSION_COOKIE_SECURE = True
     CSRF_COOKIE_SECURE = True
 
+LOCALE_PATHS = (
```

Comment:
It doesn't work without this?? Any ideas how to make it automatic? Maybe if we add a mandatory prefix to the experiment directories?

Reply:
We could do that. Like "page_"+...

Reply:
> We could do that. Like "page_"+...

No need... just add a `PROJECT_DIR` like this (e.g. below `BASE_DIR` variable)

```
PROJECT_DIR = os.path.dirname(os.path.abspath(__file__))
````
to the `settings.py` file and then set our `LOCALE_PATHS` accordingly: 
```suggestion
LOCALE_PATHS = (
    os.path.join(PROJECT_DIR, 'locale'),
)
```